diff --git a/node_modules/@stitches/react/.DS_Store b/node_modules/@stitches/react/.DS_Store
new file mode 100644
index 0000000..76d6496
Binary files /dev/null and b/node_modules/@stitches/react/.DS_Store differ
diff --git a/node_modules/@stitches/react/types/index.d.ts b/node_modules/@stitches/react/types/index.d.ts
index 8dbcc9c..a038b55 100644
--- a/node_modules/@stitches/react/types/index.d.ts
+++ b/node_modules/@stitches/react/types/index.d.ts
@@ -4,8 +4,9 @@ import type * as Config from './config'
 import type * as CSSUtil from './css-util'
 import type * as StyledComponent from './styled-component'
 
+export type { CreateStitches } from './config'
+
 export { $$PropertyValue, $$ScaleValue, $$ThemeValue } from './css-util'
-export type CreateStitches = Config.CreateStitches
 export type CSSProperties = CSSUtil.CSSProperties
 export type DefaultThemeMap = Config.DefaultThemeMap
 
