diff --git a/dist/declarations/src/index.d.ts b/dist/declarations/src/index.d.ts
index 3571ff8baf0d542d7d43191dd4bb16aa7945498d..20e174c6ff48a3cece6e14fd8bbf189e9c5d9158 100644
--- a/dist/declarations/src/index.d.ts
+++ b/dist/declarations/src/index.d.ts
@@ -7,6 +7,8 @@ export { mergeStyles } from './styles';
 export { defaultTheme } from './theme';
 export { createFilter } from './filters';
 export { components } from './components';
+export * from './Select'
+export * from './useStateManager'
 export declare type SelectInstance<Option = unknown, IsMulti extends boolean = false, Group extends GroupBase<Option> = GroupBase<Option>> = Select<Option, IsMulti, Group>;
 export type { StateManagerProps as Props } from './useStateManager';
 export { useStateManager };
